packages/editor/src/plugins/image/controls/upload-button.tsx

```diff
@@ -55,16 +58,33 @@ export function UploadButton({
         </span>
         <input
           type="file"
+          multiple={config.onMultipleUploadCallback ? true : false}
```

Comment:
Super minor, just shorter:
```suggestion
          multiple={!!config.onMultipleUploadCallback}
```